applications/video_streaming/README.md

```
@@ -0,0 +1,473 @@
+
+Download the client streaming binaries from NGC:
+
+```bash
+# Navigate to the client operator directory
+cd <your_holohub_path>/operators/video_streaming/streaming_client_enhanced
+
+# Download using NGC CLI
+ngc registry resource download-version "nvidia/holoscan_client_cloud_streaming:0.2"
+unzip -o holoscan_client_cloud_streaming_v0.2/holoscan_client_cloud_streaming.zip -d holoscan_client_cloud_streaming
+
+# Clean up
```

markdownlint-cli2 (0.18.1)

Blank line inside blockquote

(MD028, no-blanks-blockquote)